src/components/organisms/AssetActions/Compute/index.tsx

```diff
@@ -0,0 +1,317 @@
+  }, [selectedAlgorithmAsset])
+
+  const startJob = async (algorithmId: string) => {
+    try {
+      if (!ocean) return
+
+      setIsJobStarting(true)
+      setIsPublished(false)
+      setError('')
+
+      const computeService = ddo.findServiceByType('compute')
+      const serviceAlgo = selectedAlgorithmAsset.findServiceByType('access')
+
+      !hasPreviousDatasetOrder &&
+        !hasDatatoken &&
+        (await buyDT('1', (ddo as DDO).price, ddo))
+
+      !hasPreviousAlgorithmOrder &&
+        !hasAlgoAssetDatatoken &&
+        (await buyDT(
+          '1',
+          (selectedAlgorithmAsset as DDO).price,
+          selectedAlgorithmAsset
+        ))
+
+      const allowed = await ocean.compute.isOrderable(
+        ddo.id,
+        computeService.index
+      )
+
+      console.log('is dataset orderable ?', allowed)
+      const assetOrderId = hasPreviousDatasetOrder
+        ? previousDatasetOrderId
+        : await ocean.compute.orderAsset(
+            accountId,
+            ddo.id,
+            computeService.index
+          )
+
+      const algorithmAssetOrderId = hasPreviousAlgorithmOrder
+        ? previousAlgorithmOrderId
+        : await ocean.compute.orderAlgorithm(
+            algorithmId,
+            serviceAlgo.type,
+            accountId,
+            serviceAlgo.index
+          )
+
+      const output = {}
+      const respone = await ocean.compute.start(
+        ddo.id,
+        assetOrderId,
+        ddo.dataToken,
+        account,
+        algorithmId,
+        undefined,
+        output,
+        `${computeService.index}`,
+        computeService.type,
+        algorithmAssetOrderId,
+        selectedAlgorithmAsset.dataToken
+      )
+      console.log('respone', respone)
+
+      setHasPreviousDatasetOrder(true)
+      setIsPublished(true)
+    } catch (error) {
+      console.log('error found')
+      setError('Failed to start job!')
+      Logger.error(error.message)
+    } finally {
+      setIsJobStarting(false)
+    }
+  }
+
+  async function handleSubmit(
+    values: any,
+    resetForm: () => void
+  ): Promise<void> {
+    await startJob(values.algorithm)
+  }
+
+  return (
+    <>
+      <div className={styles.info}>
+        <File file={file} small />
+        <Price price={(ddo as DDO).price} conversion />
+      </div>
+
+      {type === 'algorithm' ? (
+        <Input
+          type="select"
+          name="data"
+          label="Select dataset for the algorithm"
+          placeholder=""
+          size="small"
+          value="dataset-1"
+          options={['dataset-1', 'dataset-2', 'dataset-3'].map((x) => x)}
+          // onChange={handleSelectChange}
+        />
+      ) : (
+        <Formik
+          initialValues={getInitialValues}
+          validationSchema={validationSchema}
+          onSubmit={async (values, { resetForm }) => {
+            // move user's focus to top of screen
+            window.scrollTo({ top: 0, left: 0, behavior: 'smooth' })
```

Comment:
this is unneeded here since we do not make the whole screen disappear like on publish. And here it would be a bad experience on mobile pushing user to top of screen after starting compute job